eclair-core/src/test/scala/fr/acinq/eclair/blockchain/electrum/ElectrumWalletSpec.scala

```diff
@@ -90,13 +90,13 @@ class ElectrumWalletSpec extends TestKit(ActorSystem("test")) with FunSuiteLike
 
   test("wait until wallet is ready") {
     electrumClient = system.actorOf(Props(new ElectrumClientPool(new AtomicLong(), Set(ElectrumServerAddress(new InetSocketAddress("localhost", electrumPort), SSL.OFF)))))
-    wallet = system.actorOf(Props(new ElectrumWallet(seed, electrumClient, WalletParameters(Block.RegtestGenesisBlock.hash, new SqliteWalletDb(DriverManager.getConnection("jdbc:sqlite::memory:")), minimumFee = 5000 sat))), "wallet")
+    wallet = system.actorOf(Props(new ElectrumWallet(seed, electrumClient, WalletParameters(BECH32, Block.RegtestGenesisBlock.hash, new SqliteWalletDb(DriverManager.getConnection("jdbc:sqlite::memory:")), minimumFee = 5000 sat))), "wallet")
```

Comment:
Why not reuse the way tests are defined for `ElectrumWalletSimulatedClientSpecBase` here and have BIP49 and BIP84 tests ?

Reply:
Done in 7544fb0, i also added the wallet type to the test names of parametrized test because it was getting confusing to see all those test with the same name.